samples/tests/conftest.py

```diff
@@ -125,8 +124,8 @@ def routine_id(client, dataset_id):
     routine.arguments = [
         bigquery.RoutineArgument(
             name="x",
-            data_type=bigquery_v2.types.StandardSqlDataType(
-                type_kind=bigquery_v2.types.StandardSqlDataType.TypeKind.INT64
+            data_type=bigquery.StandardSqlDataType(
```

Comment:
Let's make sure we mention these changes to Standard SQL types and enums in the migration guide. https://github.com/googleapis/python-bigquery/issues/960

Reply:
Definitely